x-pack/plugins/siem/public/management/routes.tsx

```diff
@@ -12,7 +12,7 @@ import { MANAGEMENT_ROUTING_ROOT_PATH } from './common/constants';
 /**
  * Returns the React Router Routes for the management area
  */
-export const getManagementRoutes = () => [
+export const managementRoutes = () => [
```

Comment:
What wrong with me 🤔 . I have this tendency of always naming my utility type of methods as "getters". My bad on this one - I should have follow the naming conventions seen with other plugins here.

Reply:
question on this. maybe @XavierM has thoughts:
should these be react components?

Reply:
I feel like that it does not have to be a function either, thinking about it. So component will be better

Reply:
If I understand correctly, @paul-tavares already has plans (and maybe code?) to make that change.

Reply:
managementRoutes still returns an array of components. You are saying that it should only return a component (ex. a `<Switch>...</Switch>`)?  That would be fine, but it would alter the interface to SubPlugins right?